include/wlr/util/subpixel.h

```diff
@@ -0,0 +1,8 @@
+#ifndef UTIL_SUBPIXEL_H
+#define UTIL_SUBPIXEL_H
+
+#include <wayland-server-protocol.h>
+
+const char *wlr_wl_output_subpixel_to_string(enum wl_output_subpixel subpixel);
```

Comment:
We don't use this anywhere, so we might as well remove it.

Reply:
I'm using it in https://github.com/swaywm/sway/pull/3645. I figured it would be better to put it in wlroots since other window managers would likely want to log the subpixel order.

If you still think it should be removed, I'll move it to my sway PR.

Reply:
Yeah, I'd prefer to move this to sway.